src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3532,6 +3383,204 @@
   <data name="AddPage" xml:space="preserve">
     <value>Add page</value>
   </data>
+  <data name="ProcessingItems" xml:space="preserve">
+    <value>Processing items...</value>
+  </data>
+  <data name="SpeedWithColon" xml:space="preserve">
+    <value>Speed:</value>
+  </data>
+  <data name="StatusCenter_CompressCanceled_Header" xml:space="preserve">
+    <value>Canceled compressing {0} items to "{1}"</value>
```

Comment:
Is "1" the archive name?

Reply:
Yes, “Files-src.zip”

Reply:
Can you check if it's working properly for you? It shows the location instead of the archive name for me.

Reply:
It appears to be showing the folder name and not the archive name.

Reply:
I probably know why. Please let me take a look later

Reply:
Thanks!